src/ensembl/src/styles/_mixins.scss

```diff
@@ -0,0 +1,20 @@
+@use "sass:map";
+
+$lozengeDefaults: (
+  "border-radius": 20px,
+  "padding": 6px 20px,
+  "line-height": 1
+);
+
+@mixin lozenge($lineHeight: map-get($lozengeDefaults, "line-height"), $borderRadius: map-get($lozengeDefaults, "border-radius")) {
+  display: inline-block;
+  padding: map-get($lozengeDefaults, "padding");
+  border-radius: $borderRadius;
+  line-height: $lineHeight;
+  user-select: none;
+  margin-bottom: 4px;
+
+  &:not(:last-child) {
+    margin-right: 12px;
+  }
```

Comment:
That's not the responsibility of a lozenge

Reply:
I agree, it should be removed